writer.go

```diff
@@ -327,7 +328,6 @@ func (w *LockedWriter) Write(b []byte) (n int, err error) {
 	}
 
 	n, err = w.file.Write(b)
-	w.Unlock()
```

Comment:
Good catch.

But, `defer` will add extra cost.
could you please just add `w.Unlock()` after line 321?

Reply:
Done~